airflow/sensors/base_sensor_operator.py

```diff
@@ -159,6 +160,16 @@ def _get_next_poke_interval(self, started_at, try_number):
         else:
             return self.poke_interval
 
+    def prepare_for_execution(self) -> BaseOperator:
+        task = super().prepare_for_execution()
+        # Sensors in `poke` mode can block execution of DAGs when running
+        # with single process executor, thus we change the mode to`reschedule`
+        # to allow parallel task being scheduled and executed
+        if conf.get('core', 'executor') == "DebugExecutor":
```

Comment:
```suggestion
        if conf.get('core', 'executor') == "DebugExecutor":
```
Should we use `DEBUG_EXECUTOR.DebugExecutor`?

Reply:
I have a problem with importing it as constants (MyPy complains). I prefer to leave it :(